packages/server/modules/workspaces/repositories/workspaces.ts

```diff
@@ -192,13 +193,27 @@ export const getWorkspaceCollaboratorsFactory =
       query.andWhere(DbWorkspaceAcl.col.role, role)
     }
 
+    if (cursor) {
+      query.andWhere(Users.col.name, '>', cursor)
```

Comment:
I don't think Users.name is a good cursor, there could be multiple people with the same name and that will likely break pagination. Also, I don't think you should order by Users.name, it makes more sense to show new collaborators first. Can we add a createdAt/updatedAt to workspaces_acl and order by/cursor by that?

If not, at least order by/cursor by User.createdAt (makes less sense tho).

Reply:
Also cursors should be opaque: https://graphql.org/learn/pagination/

> Especially if the cursors are opaque, either offset or ID-based pagination can be implemented using cursor-based pagination (by making the cursor the offset or the ID), and using cursors gives additional flexibility if the pagination model changes in the future. As a reminder that the cursors are opaque and that their format should not be relied upon, we suggest base64 encoding them.

Use encode/decode cursor utilities from `import { decodeCursor, encodeCursor } from '@/modules/shared/helpers/graphqlHelper'`